docs/docs/guides/providers/credentials-provider.md

```diff
@@ -82,15 +92,37 @@ callbacks: {
 }
 ```
 
+### Using a database
+
+You can also use the `jwt()` callback to interact with your database to regain some of the functionality from [more powerful providers](reference/core/providers):
+
+```js
```

Comment:
Would love some help on this pseudo-code!

Reply:
All this could live in the `authorize()` callback.

Reply:
Ah, right. Switched it up now. Does this make more sense?

Reply:
We can assume that updating jwt token in callbacks has been fixed?